cognite/client/data_classes/filters.py

```diff
@@ -944,3 +946,31 @@ def load(cls, filter_: dict[str, Any]) -> NoReturn:
 
     def _involved_filter_types(self) -> set[type[Filter]]:
         return {Not, Exists}
+
+
+class InstanceReferences(Filter):
+    """Data modeling filter which matches filters with these fully qualified instance refs.
```

Comment:
Maybe this could be rephrased using "exact match" or similar wording? Also:
```suggestion
class InstanceReferences(Filter):
    """Data modeling filter which matches instances with these fully qualified references.
```